src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -551,8 +553,12 @@ public synchronized void open() throws SFException, SnowflakeSQLException {
         connectionPropertiesMap.get(SFSessionProperty.QUERY_TIMEOUT),
         connectionPropertiesMap.get(SFSessionProperty.TRACING),
         connectionPropertiesMap.get(SFSessionProperty.PRIVATE_KEY_FILE),
-        SFLoggerUtil.isVariableProvided(
-            (String) connectionPropertiesMap.get(SFSessionProperty.PRIVATE_KEY_FILE_PWD)),
+        !Strings.isNullOrEmpty(
```

Comment:
Good catch. I missed incorporating this when resolving a merge conflict because of https://github.com/snowflakedb/snowflake-jdbc/pull/1662. I fixed that now.